### ISSUE TITLE
Embed multiple views into parent

### ISSUE THREAD

**Issue by MBuchalik** [phonegap/phonegap-plugin-multiview#57]
First of all: This is a really great example showing the "power" of cordova plugins. I really like the idea!

I was wondering if, some day, it could be possible to embed multiple (small) views within the parent webview?

Some potential use case: I am displaying external content within my app. Currently, this is done using an iframe.  But this method has some drawbacks: Imagine I want to use pinch-to-zoom gestures. As soon as I add user-scalable=no to my viewport meta tag (which is absolutely necessary in an app wanting to feel "native"), all iframes cannot be zoomed as well.
By embedding a second webview into the page, I could display external content just like it is done in every other native application. With all the benefits of having a native view instead of some iframe workaround.

Let's say I am building an email reader in PhoneGap. Have a look at how content is displayed in iOS: https://d31v04zdn5vmni.cloudfront.net/blog/wp-content/uploads/2014/09/email-aa-iphone6-plus.png
Users are able to scroll, zoom, ... this view smoothly. Being able to implement some similar function into my PhoneGap app would bring endless possibilities with native performance instead of some JS hacks :smiley:

**Comment by Firzenizer** [comment]
This is what we would need in our application too. I was excited, when I saw this plugin was released. This plugin was supposed to fix our problems of not being able to use iframes.

We would need to be able to control the size of the 2nd web view. Also ability to close the 2nd web view from the primary web view is required.

We are trying to embed a web page, that doesn't allow iframes so 2nd web view is our only chance. But in the current state, it doesn't serve the purpose for us.
